src/main/java/seedu/socket/logic/commands/UnassignCommand.java

```diff
@@ -54,17 +54,15 @@ public CommandResult execute(Model model) throws CommandException {
         Project projectToUnassign = lastShownList.get(index.getZeroBased());
         Set<Person> members = projectToUnassign.getMembers();
         Set<Person> newMembers = new HashSet<>();
-        Boolean isMember = false;
+        Boolean isMember = projectToUnassign.hasMember(name);
```

Comment:
Perhaps consider simply having `if (!projectToUnassign.hasMember(name))`? Not really an issue atm, it's just that the variable is used once and nowhere else, let me know your thoughts; if you feel there's no need I'll go ahead and merge

Reply:
sure i'll do that